src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -679,6 +680,7 @@ bool Dataloader::_load_map_dir(GameManager& game_manager) const {
 	static constexpr std::string_view default_continent = "continent.txt";
 	static constexpr std::string_view default_adjacencies = "adjacencies.csv";
 	static constexpr std::string_view default_region = "region.txt";
+	static constexpr std::string_view default_region_colours = "colors.txt"; /* File from interface_directory */
```

Comment:
The region colours filepath is hardcoded, it can't be customised in Vic2's `default.map`. Move this line to the bottom of this block (next to `climate_file`) and rename it from `default_region_colours` to just `region_colours`.